src/views/Batches/batchComponents/BatchForm.jsx

```diff
@@ -121,6 +122,13 @@ const BatchForm = (props) => {
         // otherwise return only those status that are applicable to all
         return status['applicable-to'] === 'All';
       });
+
+      let modeOfPayment = data.mode_of_payment;
+      setPaymentOptions(modeOfPayment.map(mode_of_payment => ({
```

Comment:
can do directly this instead of defining an intermediary variable, since it's only needed at one place.
```suggestion
      setPaymentOptions(data.mode_of_payment.map(mode_of_payment => ({
```